crates/bevy_ui/src/node_bundles.rs

```diff
@@ -207,26 +205,6 @@ pub struct TextBundle {
 
 #[cfg(feature = "bevy_text")]
 impl TextBundle {
-    /// Create a [`TextBundle`] from a single section.
```

Comment:
As discussed in discord. 

The removal of these ergonomics is a huge loss. both on `TextBundle` and on `Text` and should be addressed before 0.15 if this PR is to land